tests/pytests/test_info_modules.py

```diff
@@ -481,32 +481,33 @@ def test_redis_info_modules_vecsim():
   env = Env(moduleArgs='WORKER_THREADS 2 MT_MODE MT_MODE_FULL')
   env.expect(config_cmd(), 'SET', 'FORK_GC_CLEAN_THRESHOLD', '0').ok()
   set_doc = lambda: env.expect('HSET', '1', 'vec', '????????')
-  field_infos = lambda: [to_dict(env.cmd(debug_cmd(), 'VECSIM_INFO', f'idx{i}', 'vec')) for i in range(1, 4)]
+  get_field_infos = lambda: [to_dict(env.cmd(debug_cmd(), 'VECSIM_INFO', f'idx{i}', 'vec')) for i in range(1, 4)]
```

Comment:
why not to use `FT.DEBUG WORKER_THREADS DRAIN` as we do in master?

Reply:
We don't have it in 2.8...